x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -78,7 +78,9 @@ expire.
 [[security-api-create-api-key-example]]
 ==== {api-examples-title}
 
-The following example creates an API key:
+The following example creates an API key.
+
+NOTE: If your node has `xpack.security.http.ssl.enabled` set to `true`, then you must specify `https` when creating your API key.
```

Comment:
Even though this is true, there is no where we use `http` or `https` explicitly in this page to create API keys. So this note feels out of place. I think we could revert it back to its original place and change it to something like:

```
If your node has `xpack.security.http.ssl.enabled` set to `true`, then you must specify `https` in the following command.
```